src/kirby/components/avatar/avatar.component.scss

```diff
@@ -70,6 +71,22 @@ $badge-diameter: 16px;
   }
 }
 
+:host(.xs) {
+  .avatar {
+    width: $image-diameter-xsmall;
+    height: $image-diameter-xsmall;
+    border-radius: 8px;
```

Comment:
Why is this different from the rest of the sizes? (`50%`)

Reply:
The issue #444 description says "We have an avatar with an icon that is not shaped as a circle but as a round rectangle" and looking in zeplin the border-radius is set to 8px